README.md

```diff
@@ -1,4 +1,4 @@
-### Math Magicians
+### New Math Magicians
 
```

Comment:
- [ ] I recommend adding a project screenshot, Github repo issues page link to Readme file to make it more informative.

Reply:
Thank you for the review. I have implemented your suggestion